### PR TITLE
[BHM-AL-2024] Organizer updates

### PR DESCRIPTION
**Pull request by zonywhoop**
* Adding new organizer [Elizabeth Anderson] - core team emailed
* Updating all organizer info and adding pics
* Updating welcome page and fixing city definition

### CODE REVIEW

File: content/events/2024-birmingham-al/welcome.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ Description = "devopsdays Birmingham, AL 2024"
...
 </div>
-
-<!-- <div class = "row">
+<div class = "row">
   <div class = "col-md-2">
     <strong>Propose</strong>
   </div>
   <div class = "col-md-8">
-    {{< event_link page="propose" text="Propose a talk!" >}}
+   <a href="https://talks.devopsdays.org/devopsdays-birmingham-al-2024/cfp">Propose a talk!</a>
```
Comment (glasnt) [inline content/events/2024-birmingham-al/welcome.md:78]: This change shouldn't be needed, as the cfp_link should already populate here.

File: content/events/2024-birmingham-al/welcome.md [hunk 2]
```diff
@@ -124,9 +124,15 @@ Description = "devopsdays Birmingham, AL 2024"
...
-
- <a rel="me" target="_blank" href="https://social.devopsdays.org/@devopsdaysbham"><span class="btn btn-sm rounded-pill btn-info"><i class="fab fa-mastodon fa-md" ></i> Follow @devopsdaysbham@devopsdays.org</span></a>
\ No newline at end of file
+<div class = "row" style="padding-top: 10px">
+  <div class = "col-md-3">
+    <a rel="me" target="_blank" href="https://social.devopsdays.org/@devopsdaysbham"><span class="btn btn-sm rounded-pill btn-info" style="background-color: #605FEE"><i class="fab fa-mastodon fa-md" ></i> Follow @devopsdaysbham@devopsdays.org</span></a>    
+  </div>
+  <div class = "col-md-2">
+    <a rel="me" target="_blank" class="twitter-follow-button" href="https://twitter.com/devopsdaysbham" data-size="large">Follow @devopsdaysbham</a>
+    <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```
Comment (glasnt) [inline content/events/2024-birmingham-al/welcome.md:133]: Are you intentionally adding an embed of an X feed on your front page?

**Comment by glasnt** [comment]
It looks like you need to sync your fork, there are many more commits than intended here.

File: data/events/2024-birmingham-al.yml [hunk 3]
```diff
@@ -52,21 +52,28 @@ team_members: # Name is the only required field for team members.
...
-    employer: "Hedgehog"
+    image: "mikhail.jpg"
   - name: "Ed McLain"
     linkedin: "https://www.linkedin.com/in/edwardmclain/"
     employer: "Nibuild"
     twitter: "zonywhoop"
     github: "zonywhoop"
-
+    image: "ed-mclain.jpg"
+  - name: "Elizabeth Anderson"
```
Comment (glasnt) [inline data/events/2024-birmingham-al.yml:71]: Elizabeth has been added to the mailing list

File: content/events/2024-birmingham-al/welcome.md (hunk 1 above)
Comment (zonywhoop) [reply content/events/2024-birmingham-al/welcome.md]: Fixed this up to use event_link with cfp_link.

File: content/events/2024-birmingham-al/welcome.md (hunk 2 above)
Comment (zonywhoop) [reply content/events/2024-birmingham-al/welcome.md:133]: We are - the use of the data-size="large" property was not in the shortcode but it's needed to match the button size we have for mastadon.

**Comment by zonywhoop** [comment]
Noting the additional commits, I think that was a goof on our side. Our fork is in sync but I think I need delete the erroneous commits and do a full re-sync once this PR is applied. I'm thinking it was caused by a multiple commits getting squashed then merged from a local pull along the way.

**Pull request merged**